app/modules/configuration/resources.py

```diff
@@ -34,6 +34,7 @@
     'email_service_password',
     'email_default_sender_email',
     'email_default_sender_name',
+    'relationship_type_roles',
```

Comment:
We talked in discord about having social group types as site settings. 
Would social group types be different to relationship types. 
Is there any scope for sharing types between relationship and social group or are they completely different types.